update contributor instructions for fork-based workflow (#638)
leanprover-community/leanprover-community.github.io: templates/contribute/index.md

* Use [Zulip](https://leanprover.zulipchat.com/) to
   discuss your contribution before and while you are working on it.

> * Please include your GitHub username in your request and add this username to your Zulip profile, using [the personal settings panel](https://leanprover.zulipchat.com/#settings/profile).

Author: Ah, you're right. I just copied + pasted this from the earlier version of the text without reading it too carefully.

> * Please include your GitHub username in your request and add this username to your Zulip profile, using [the personal settings panel](https://leanprover.zulipchat.com/#settings/profile).

Author: ```suggestion
* Create a GitHub account and add your GitHub username to your Zulip profile, using [the personal settings panel](https://leanprover.zulipchat.com/#settings/profile).
```

On the main page for a PR, on the right-hand side,
there should be a sidebar with panels "reviewers", "assignees", "labels", etc.
Click on the "labels" header to add or remove labels from the current project.
(Labels can only be edited by "GitHub collaborators", which is approximately the same as "people who have write access".) Anyone can edit the labels by writing the following commands in a comment on the PR (each on its own line):
- `awaiting-author` will add the **"awaiting-author"** label
- `-awaiting-author` will remove the **"awaiting-author"** label
- `WIP` will add the **"WIP"** label

> - `-WIP` will remove the **"WIP"** label

Author: cf. https://github.com/leanprover-community/mathlib4/pull/25723
